lib/resources/r4/financial_transaction.yaml

```diff
@@ -248,9 +247,12 @@ fields:
     required: 'Yes'
     type: Extension
     description: The specific target that the payment or adjustment should be posted to. This extension is nested under a <code>financial-transaction-allocation</code> extension.
-    note:
-      - The <code>valueReference.reference</code> must be an Encounter reference.
-      - The URL for this extension is <code>target</code>.
+    note: >
+      <ul>
+        <li>The <code>valueReference.reference</code> must be either an Encounter or Account reference.</li>
+        <li>The Account reference must be of the guarantor-balance type.</li>
```

Comment:
I think it reads better to say: `The Account reference must be of the <code>guarantor-balance</code> type.`

Reply:
Let's also add a conditional, i.e.
```
If there reference is of type Account, it must be a guarantor-balance account type.
```

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/794/commits/4ca97ce64ea00939f679e4d26c2073012dab3271#diff-fbd410f5e96863ae93ac388f712817417b0247695fdb087032c433dd619b4239R253

Reply:
Left a nit on the commit - https://github.com/cerner/fhir.cerner.com/commit/4ca97ce64ea00939f679e4d26c2073012dab3271#r64522308

Reply:
Reminder to update the screenshot after making the nit update as well :)

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/794/commits/3be844d058fffd1b72d105d9ae0aa19e7689d466#diff-fbd410f5e96863ae93ac388f712817417b0247695fdb087032c433dd619b4239R253